app/queue.py

```diff
@@ -22,6 +22,9 @@ class Buffer(Enum):
     def name(self, suffix=None):
         return f"{self.value}:{suffix}" if suffix else self.value
 
+    def get_inflight_name(self, receipt: UUID = uuid4(), suffix: str = "") -> str:
```

Comment:
Can we please pass in `None` to the `suffix` parameter?

Reply:
Sone